platform/vs/docker-syncd-vs.mk

```diff
@@ -12,3 +12,5 @@ $(DOCKER_SYNCD_BASE)_DBG_DEPENDS += $(SYNCD_VS_DBG) \
                                 $(LIBSAIVS_DBG)
 
 $(DOCKER_SYNCD_BASE)_RUN_OPT += -v /host/warmboot:/var/warmboot
+
+$(DOCKER_SYNCD_BASE)_BASE_IMAGE_FILES += tc-syncd:/usr/bin/tc-syncd
```

Comment:
why do we need to expose tc in syncd to the base image?

Reply:
As indicated earlier, this was added only for debugging. Now removed.